src/Generator/Generators/ExtensionMethods.cs

```diff
@@ -23,5 +24,30 @@ public static Interop.CallingConvention ToInteropCallConv(this CallingConvention
 
             return Interop.CallingConvention.Winapi;
         }
+
+        public static bool CheckClassIsStructible(Class @class, Driver Driver)
+        {
+            if (!@class.IsStruct)
+                return false;
+            if (@class.IsInterface || @class.IsStatic || @class.IsAbstract)
+                return false;
+            if (@class.Methods.Any(m => m.IsOperator))
```

Comment:
Why can't it be a struct if there are any operators? As far as I know C# structs also support operators.